stylelint-polaris/index.js

```diff
@@ -309,16 +331,9 @@ const stylelintPolarisCoverageOptions = {
         ],
       },
     },
-    'stylelint-polaris/global-disallowed-list': [
-      / \* \$/,
-      // Legacy custom properties
-      /--p-override-none/,
-      /--p-override-one/,
-      /--p-override-zero/,
-      /--p-non-null-content/,
-    ],
   },
-  mediaQueries: {
+  breakpoints: {
```

Comment:
`mediaQueries` may be more appropriate as this category can lint more than Polaris breakpoints e.g. [media types](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#media_types), [media conditions](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#media_features), [media conditions/operators](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#logical_operators), etc. Thoughts?